www/nginx/Makefile

```diff
@@ -1,8 +1,7 @@
 PLUGIN_NAME=          nginx
-PLUGIN_VERSION=       0.4
+PLUGIN_VERSION=       1.1
```

Comment:
this makes no sense...

Reply:
sure it does 0.4 is 1.0 for 18.1 series. 1.0 should have been released with 18.7.1

Reply:
nginx was never on 18.1

Reply:
@fichtner then some plans got terribly wrong...

Reply:
we can release 1.0 in 18.7.2 if you want, but you will have to ask or let me know explicitly. you need to be on top of this ;)